.npmignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 src
+example/src
```

Comment:
npmignore follows same rules as gitignore. prior `src` will cover `example/src`, so this is not needed